### DISCUSSION TITLE
Is there a way to format duration (in milliseconds) into minutes, seconds, milliseconds ?

### DISCUSSION THREAD

**Discussion by ningji** [js-fns/date-fns#3597] [Q&A]
This is the output format i'm expecting 
milliseconds => { hours, minutes, seconds, milliseconds } 

so i can display
```The duration of scripts: 0m 0s 35ms```

Do we have a buildin function to do so in date-fns ?
Thanks !

**Comment by LinThitHtwe** [comment]


**Reply by ErwanLeroux** [reply]
milliseconds does not exist in Duration

export interface Duration {
  /** The number of years in the duration */
  years?: number;
  /** The number of months in the duration */
  months?: number;
  /** The number of weeks in the duration */
  weeks?: number;
  /** The number of days in the duration */
  days?: number;
  /** The number of hours in the duration */
  hours?: number;
  /** The number of minutes in the duration */
  minutes?: number;
  /** The number of seconds in the duration */
  seconds?: number;
}
